src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_virtual_network.py

```diff
@@ -176,7 +176,7 @@ def _create_vnet_subnet_delegation(nw_client, resource_group, vnet_name, subnet_
                                                                              location=location,
                                                                              address_space=AddressSpace(
                                                                                  address_prefixes=[
-                                                                                     vnet_address_pref])))
+                                                                                     vnet_address_pref]))).result()
```

Comment:
Another solution proposed by @zhoxing-ms is to supply `polling=False` to `nw_client.virtual_networks.begin_create_or_update` so that `LROPoller` is created with `NoPolling`:

https://github.com/Azure/azure-sdk-for-python/blob/ad5308d9befefdc0cdc61c4a47a3329fd874b8a0/sdk/network/azure-mgmt-network/azure/mgmt/network/v2020_08_01/operations/_virtual_networks_operations.py#L340-L341

```py
if polling is True: polling_method = ARMPolling(lro_delay, lro_options={'final-state-via': 'azure-async-operation'}, path_format_arguments=path_format_arguments,  **kwargs)
elif polling is False: polling_method = NoPolling()
```
This stops SDK from polling the result. The Virtual Network creation will then be **superseded/canceled** by Subnet creation.

### Pros

- This is faster as it doesn't care about the result of Virtual Network creation.

### Cons

- This leaves the result of Virtual Network creation unchecked. If the Virtual Network creation actually fails, the error will be reported at 
  https://github.com/Azure/azure-cli/blob/95d3346aeab751820eb5c09be588586cac409890/src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_virtual_network.py#L188-L189
  making troubleshooting difficult.
- This depends on the service's ability to allow one operation to be superseded by another. In many other services like Storage, this is not always the case - we can't create a container before Storage Account creation finished.